Sming/Libraries/DS18S20/ds18s20.cpp

```diff
@@ -255,3 +259,12 @@ uint8_t DS18S20::GetSensorsCount()
 
 }
 
+void DS18S20::RegisterEndCallback(void (*Callback)())
```

Comment:
@Ogyb Can you replace the callback with Delegates. This way we can have the old C style callback and also method calling from objects.